corehq/apps/auditcare/tests/test_tasks.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+
+from ..models import AccessAudit, NavigationEventAudit
+from ..tasks import copy_events_to_sql
+from .testutils import AuditcareTest, delete_couch_docs, save_couch_doc
+
+
+class TestCopyEventsToSQL(AuditcareTest):
+    username = "audit@care.com"
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        username = "couch@test.com"
+        cls.couch_ids = [
+            save_couch_doc(
+                "NavigationEventAudit",
+                username,
+                event_date=datetime(2021, 2, 1, 2).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                description='User Name',
+                extra={},
+                headers={
+                    'REQUEST_METHOD': 'GET',
+                },
+                ip_address='10.1.2.3',
+                request_path='/a/delmar/phone/restore/?version=2.0&since=...',
+                session_key='abc123',
+                status_code=200,
+                user_agent='okhttp/4.4.1',
+                view_kwargs={'domain': 'delmar'},
+                view='corehq.apps.ota.views.restore',
+            ),
+            save_couch_doc(
+                "AccessAudit",
+                username,
+                event_date=datetime(2021, 2, 1, 3).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                access_type='login',
+                description='Login Success',
+                failures_since_start=None,
+                get_data=[],
+                http_accept='text/html',
+                ip_address='10.1.3.2',
+                path_info='/a/delmar/login/',
+                post_data=[],
+                session_key='abc123',
+                user_agent='Mozilla/5.0',
+            ),
+            save_couch_doc(
+                "NavigationEventAudit",
+                username,
+                event_date=datetime(2021, 2, 1, 5).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                description='User Name',
+                extra={},
+                headers={
+                    'REQUEST_METHOD': 'GET',
+                },
+                ip_address='10.1.2.3',
+                request_path='/a/sandwich/phone/restore/?version=2.0&since=...',
+                session_key='abc123',
+                status_code=200,
+                user_agent='okhttp/4.4.1',
+                view_kwargs={'domain': 'sandwich'},
+                view='corehq.apps.ota.views.restore',
+            ),
+        ]
+
+    @classmethod
+    def tearDownClass(cls):
+        delete_couch_docs(cls.couch_ids)
+        super().tearDownClass()
+
+    def test_copy(self):
```

Comment:
Looks like this test copies all of the test data. What do you think of adding a couple test records (at least one before and one after the copied date range) and verify that those records are not copied?

The purpose of that is to verify that your query range parameters are working as expected.